marklogic-junit/examples/simple-dhf4/README.md

```diff
@@ -0,0 +1,33 @@
+This project shows an example of using both marklogic-junit and marklogic-unit-test within a 
+Data Hub Framework version 4 project. 
+
+To try this out locally, first initialize the DHF project:
+
+    ./gradlew hubInit
+    
+Then add the following lines to gradle.properties:
+
```

Comment:
I think it would be a good idea to add a line of doc here just to remind the user that the user below will be used in the test requests and that it and the password needs to be set.. maybe with a hint towards the `data-hub-user`